src/ReactWrapper.js

```diff
@@ -294,6 +295,7 @@ export default class ReactWrapper {
    * @returns {ReactWrapper}
    */
   simulate(event, ...args) {
+    event = mapNativeMouseEvent(event);
```

Comment:
this approach seems both brittle and only will work on mouse events. There's lots of other kinds of events, like "dragstart", "touchstart", "keypress", etc that this could impact.

What about an explicit whitelist - just an object mapping lowercase names to react-cased ones?